option optimize_for = LITE_RUNTIME;

message RebelExploitRewardRequest
{
 required uint32 id = 1;
}
